scripts/kmsClientIdAuthorizer.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+const jwt = require("jsonwebtoken");
+const rp = require("request-promise-native");
+
+const KMS = new AWS.KMS({region: 'eu-west-1'});
+
+const defaultConfig = {
+    domain: "cimpress.auth0.com",
+    audience: "https://api.cimpress.io/"
+};
+
+class KmsBasedAuth0Authenticator {
```

Comment:
This file is inspired from `cimpress-translations-webpack-plugin` but it's simplified.